Tools/CISupport/ews-build/events.py

```diff
@@ -147,11 +197,19 @@ def buildFinished(self, key, build):
         builder = yield self.master.db.builders.getBuilder(build.get('builderid'))
         builder_display_name = builder.get('description')
 
+        patch_id = self.extractProperty(build, 'patch_id')
+
+        if self.extractProperty(build, 'github.number'):
```

Comment:
Are we using 'github.number' as the main identifier to decide whether it is a PR or a patch based build? Whatever we are using, let's make sure to use the same identifier through-out the ews code.

Reply:
Yes, that's what I've been keying off of. I often check more that just `github.number`, but I try to make that the primary check.